TypeCobol.Test/Parser/Programs/Cobol85/UnsupportedKeywords.rdzMix.txt

```diff
@@ -0,0 +1,16 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 WS-FOO PIC X.
+       01 WS-BAR PIC X.
+      *EXHIBIT, NAMED and CHANGED are not reserved keywords
+       01 EXHIBIT PIC X.
+       01 CHANGED PIC X.
+       01 NAMED PIC X.
+       PROCEDURE DIVISION.
+      *EXHIBIT statement is not supported
+Line 13[20,24] <27, Error, Syntax> - Syntax error : 'EXHIBIT' is not supported
```

Comment:
Because this is a E-I specific feature, I would prefer to use the following message :
"E-I compilation doesn't allow use of EXHIBIT instruction."

Reply:
Is this checked only for E-I environment?

Reply:
No the error will be displayed regardless of the compilation configuration (E-I or regular). It is not a check because the keyword is not known, it is an ANTLR parsing error. In this PR, we adjust the message so as to explain to the end user that the keyword/statement is not supported.
I don't think this should be considered as E-I specific because `EXHIBIT` is not part of IBM cobol specs and never has been supported in our parser. To reject it in E-I environment only, we would have to support it in the first place.

Reply:
OK it makes sense.